books/free-programming-books-hi.md

```diff
@@ -39,6 +39,7 @@
 ### Java
 
 * [Java \| Hindi](https://www.learnhindituts.com/java) - LearnHindiTuts.com
+* [Java \| Hindi](https://ehindistudy.com/2022/10/14/java-pdf-book-hindi/) - ehindistudy.com
```

Comment:
I cannot find a resource with that nome on this page.

Reply:
That's because, Hindi study material is scarcely available, and to disseminate information websites tend to keep the links hidden from publishers. I know this is wrong, so what would you suggest in this scenario?